test/document_test.rb

```diff
@@ -30,7 +30,7 @@ def add_to_foo key, value
 end
 
 class WithTime < Dolly::Document
-  property :created_at
+  property :created_at, default: -> {DateTime.now}
```

Comment:
This fixes a test that should have been failing the entire time, but was passing until I made my changes.